model/src/main/kotlin/Provenance.kt

```diff
@@ -66,11 +66,9 @@ data class RepositoryProvenance(
     val vcsInfo: VcsInfo,
 
     /**
-     * The original [VcsInfo] that was used to download the source code. It can be different to [vcsInfo] if any
-     * automatic detection took place. For example if the original [VcsInfo] does not contain any revision and the
-     * revision was automatically detected by searching for a tag that matches the version of the package there
-     * would be no way to match the package to the [Provenance] without downloading the source code and searching
-     * for the tag again.
+     * The [originalVcsInfo] is is equal to the [meta-data of the package][Package.vcsProcessed]. Thus, it has an empty
+     * resolved revision (as of 835b197). All attributes of [originalVcsInfo], except for the resolved revision, equal
```

Comment:
Please reword a bit to be able to drop the reference to a concrete commit SHA1. Or maybe just drop the commit completely?